remote-api/src/main/java/org/alfresco/rest/api/model/rules/CompositeCondition.java

```diff
@@ -50,7 +52,7 @@
      * @param actionConditions - list of {@link ActionCondition} service POJOs
      * @return {@link CompositeCondition} REST model
      */
-    public static CompositeCondition from(final List<ActionCondition> actionConditions)
+    public static CompositeCondition from(final List<ActionCondition> actionConditions, final NamespaceService namespaceService)
```

Comment:
I think we should introduce a mapping "layer" for mapping from/to service and API models instead of mapping inside the model classes.